sql/parser/eval.go

```diff
@@ -316,6 +319,13 @@ var binOps = map[binArgs]binOp{
 	},
 }
 
+func inDays(d time.Duration) (DDate, error) {
+	if d%(24*time.Hour) != 0 {
```

Comment:
I've taken out support for arithmetic between a date and a duration and supplanted it with arithmetic between a date and an int.